libs/database/src/lib/services/content.service.ts

```diff
@@ -247,6 +247,39 @@ export class ContentService {
     return content.save();
   };
 
+  /**
+   * Delete Content from orginal post and author
+   * @param {string} originalPostId
+   * @param {string} authorId
+   * @returns {null}
+   */
+  deleteContentFromOriginalAndAuthor = async (
+    originalPostId: string,
+    authorId: string
+  ) => {
+    this.logger.log('get content for delete.');
+    const content = await this._contentModel
+      .findOne({
+        'author.id': mongoose.Types.ObjectId(authorId),
+        'originalPost._id': mongoose.Types.ObjectId(originalPostId)
+      })
+      .exec();
+
+    if (content) {
```

Comment:
```suggestion
    if (content) return;
```

please use guard return to reduce block scope